.gitignore

```diff
@@ -46,7 +46,7 @@ playground.xcworkspace
 #
 # Xcode automatically generates this directory with a .xcworkspacedata file and xcuserdata
 # hence it is not needed unless you have added a package configuration file to your project
-# .swiftpm
+.swiftpm
```

Comment:
I think we should also uncomment lines 42, 43, and 44 above.

Reply:
Done 👍